sherpa/fit.py

```diff
@@ -24,8 +24,10 @@
 
 from functools import wraps
 
+import numpy as np
```

Comment:
Is there a reason why you only moved some of the numpy imports (if. float32, finfo) into the np namespace but left the other imports from numpy in place?  Would it be better to be consistent?

Reply:
I'm not aware we have a "standard" for our imports, so I felt it was better to leave as is rather than make wholesale changes. I do occasionally try to upgrade to the 'import numpy as np' style to save those precious three characters, but in this particular case I felt it was risky to go through and find all the "naked" symbols (e.g. `arange` and `abs`) and convert them to their "namespaced version" (e.g. `np.arange`, `np.abs`, ...). I think it would be a good thing, but in a separate PR to be done later.